src/Apps/Order/Routes/__tests__/Shipping.jest.tsx

```diff
@@ -199,21 +204,295 @@ const testMe: ShippingTestQuery$rawResponse["me"] = {
   },
 }
 
+const pushMock = jest.fn()
+let isCommittingMutation
+const relayEnv = createMockEnvironment()
+
+// This config extracted so it can be used with both enzyme (setupTestWrapper)
+// and react-testing-library (setupTestWrapperTL) tests.
+// We need to pass in our own relayEnv to both <MockBoot /> and
+// renderWithRelay so that downstream QueryRenderers can use the test
+// environment. Otherwise SystemContext will create a new [real] environment.
```

Comment:
#superminor - You can write multiline comments in VSCode by typing 
```
/** <hit enter>
```

Reply:
yeah, sometimes you start out thinking you are writing a single line comment and things just get out of hand.